components/nimbus/src/tests/mod.rs

```diff
@@ -0,0 +1,12 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+mod test_enrollment;
+mod test_enrollment_bw_compat;
+mod common;
+
+use nimbus::{
+    error::Result, AppContext,
+};
+use serde_json::{json, Value};
```

Comment:
We should move those into the individual test files that need them 😄 and replace `nimbus` with `crate` - since those are unit tests, they are internal modules to the crate, so we reference the crate using `crate::`